### DISCUSSION TITLE
Issue with pushing code to github

### DISCUSSION THREAD

**Discussion by Nikhil-dath** [community/community#121553] [Repositories]
### Select Topic Area

General

### Body

hey folks, previously my friend was using my github account he created his own profile and used it for a while so later i created my account and started using it but i don't know what happend i am encountering an error whenever i tried to push the code to the github i given the error below
"remote: Permission to Nikhil-dath/nikhil-.github.io.git denied to pavan-gangadhar-001.
fatal: unable to access 'https://github.com/Nikhil-dath/nikhil-.github.io.git/': The requested URL returned error: 403"
 my friend forgot's his password ,so if anyone understand the situation ,pls help me have tried  various solution to resolve this error

**Comment by gurjeetsinghvirdee** [comment]
Hey @Nikhil-dath 

Ask your friend to fork your repository, then clone it. He can't directly push the changes to your repository as you are the owner, but you can do one thing: invite him as a collaborator in your repo so he can directly make changes by just cloning the repo.

And if your friend forgot his password, then ask him to reset it. Simple!
